src/auth/auth.service.ts

```diff
@@ -230,4 +253,33 @@ export class AuthService {
       true,
     );
   }
+
+  public async resetPasswordEmail(
+    dto: EmailDto,
+    domain?: string,
+  ): Promise<IMessage> {
+    const user = await this.usersService.uncheckedUserByEmail(dto.email);
+
```

Comment:
before sending an email to reset the password, we check first if the user has a valid email or not